### DISCUSSION TITLE
Simplify HTML structure on ObjectElement

### DISCUSSION THREAD

**Discussion by frerotg** [vueform/vueform#331] [Questions]
Hello,

Is it a way to simplify my ObjectElement  generated html. I would like to use it but without 21321321 useless(in my case) tag container.
A simple div will be usefull for me...

I thinking about a CustomObjectElement but this object are dependent of sub component like ElementLayout which is use by others components...

Thanks you for your response.

**Comment by adamberecz** [comment]
You can use [`inline`](https://vueform.com/reference/object-element#option-inline). If you need customization beyond that, I would recommend creating eg. a `ObjectElement_simple` [alternative view](https://vueform.com/docs/styles-and-layout#alternative-views).
